test-requirements.txt

```diff
@@ -1,9 +1,11 @@
 -r mypy-requirements.txt
 -r build-requirements.txt
 attrs>=18.0
+black==22.1.0
```

Comment:
I guess these need comments pointing to the pre-ci thing too.

Reply:
Done